packages/DatePicker/src/DatePicker.js

```diff
@@ -165,6 +170,7 @@ function DatePicker(props) {
     } else {
       setConfirmationResult("");
       setHasParsingError(true);
+      onDateParseError();
```

Comment:
Do you need any information for example `inputtedString`?

Reply:
Hmm, not in my case but perhaps someone else _could_? I'm don't think they would but i can pass it as well.

Reply:
I've changed just in case someone may want to display it somehow